packages/shared/src/graphql/users.ts

```diff
@@ -542,14 +543,18 @@ export const USER_STREAK_RECOVER_QUERY = gql`
       canRecover
       cost
       oldStreakLength
+      regularCost
     }
   }
 `;
 
 export const USER_STREAK_RECOVER_MUTATION = gql`
-  mutation RecoverStreak {
-    recoverStreak {
+  mutation RecoverStreak($cores: Boolean!) {
+    recoverStreak(cores: $cores) {
```

Comment:
Should we just set this to sending a static value of true? Since webapp is going to be latest anyway and we're using this to avoid compat issues if I understand correctly.

Reply:
yeah, actually better 👌

Reply:
Added in e4195bb9109a8f6c43534668c332cf9876471233